rosbag2_storage/test/rosbag2_storage/test_constants.hpp

```diff
@@ -22,6 +22,7 @@ constexpr const char * const READ_WRITE_PLUGIN_IDENTIFIER = "ReadWritePlugin";
 constexpr const char * const READ_ONLY_PLUGIN_IDENTIFIER = "ReadOnlyPlugin";
 constexpr const char * const DUMMY_FILEPATH = "/path/to/storage";
 constexpr const uint64_t MAX_BAGFILE_SIZE = 0;
+constexpr const uint64_t MIN_SPLIT_FILE_SIZE = INT64_MAX;
```

Comment:
Should be `UINT64_MAX` since type is unsigned.

since this is C++,
```c++
MIN_SPLIT_FILE_SIZE = std::numeric_limits<uint64_t>::max();
```

Reply:
Changed to `UINT64_MAX`. Using these constants to keep it consistent with other packages in the project.